# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../nufw/nufw-2.4.3-mysql-5.5-header.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- nufw-2.4.3/src/nuauth/modules/log_mysql/log_mysql.h.orig	2011-03-25 01:22:56.872070854 +0100
+++ nufw-2.4.3/src/nuauth/modules/log_mysql/log_mysql.h	2011-03-25 01:23:14.612071722 +0100
@@ -17,7 +17,7 @@
 */
 
 #include <sys/time.h>
-#include <mysql/mysql.h>
+#include <mysql.h>
 
 
 #define MYSQL_SERVER "127.0.0.1"
